cli/utils/i18n.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2016 IBM Corporation
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package utils
```

Comment:
The CLI compiles the locales before building the binary which allows copying/moving the binary to any folder. Since the existing code does not provide this functionality, it can not be reused.

If we really want to add this functionality for other packages, we should do it later in a different PR.